plugins/generator-1.15.2/forge-1.15.2/procedures/entity_set_oxygen.java.ftl

```diff
@@ -0,0 +1 @@
+((LivingEntity)${input$entity}).setAir((int)${input$air});
```

Comment:
As I wrote previously, no need to cast here and you also shouldn't. The method is from Entity so why casting to LivingEntity. Please check others too if it applies to other codes.